### PR TITLE
[WIP] Admin/organiser events CRUD

### LINKED ISSUE #99: As an organizer I can email the entire list of participants.
Implementation details:

- UI: /dashboard/events will need buttons to email all subscribed users after a confirm dialog has been clicked through
- Sending the email: we already have a `sendEventInvite` mutation https://github.com/freeCodeCamp/chapter/blob/8ef1db39c5137e5dd49f5dd974107b57f3eda231/server/controllers/Events/resolver.ts#L259 
- It should be possible to reuse most of that, but allow the emails sent to vary.

### PR DESCRIPTION
**Pull request by Zeko369**
Admin/organiser can
- create events
- cancel event
- message people RSVPed to the events
- update events

closes #99, #100

### CODE REVIEW

File: pages/dashboard/events/[id].tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+
+const Event: React.FC = () => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return <h1>You&apos;re on event: {id}</h1>;
```
Comment (timmyichen) [inline pages/dashboard/events/[id].tsx]: You can just use an apostraphe in `'` :P

File: pages/dashboard/events/index.tsx [hunk 2]
```diff
@@ -23,7 +24,7 @@ const Events: React.FC = () => {
     <>
       <Grid container spacing={2}>
         <Grid item xs={12}>
-          <button>Add new</button>
+          <Link href="/dashboard/events/create">Create a event</Link>
```
Comment (timmyichen) [inline pages/dashboard/events/index.tsx]: nit: Create **an** event

File: client/components/DashboardEvent.tsx [hunk 3]
```diff
@@ -11,11 +12,18 @@ interface IDashboardEventProps {
...
 const DashboardEvent: React.FC<IDashboardEventProps> = ({ event, loading }) => {
   return (
-    <Card style={{ marginTop: '12px' }} key={`event-${event.id}`}>
+    <Card style={{ marginTop: '12px' }}>
       <ProgressCardContent loading={loading}>
-        <Typography gutterBottom variant="h5" component="h2">
-          {event.name}
-        </Typography>
+        <Link
+          href="/dashboard/events/[id]"
```
Comment (timmyichen) [inline client/components/DashboardEvent.tsx]: `[id]`?

File: pages/dashboard/events/[id].tsx (hunk 1 above)
Comment (Zeko369) [reply pages/dashboard/events/[id].tsx]: Me: '
Eslint: 😡
hahahahah

File: pages/dashboard/events/index.tsx (hunk 2 above)
Comment (Zeko369) [reply pages/dashboard/events/index.tsx]: 🙈

File: client/components/DashboardEvent.tsx (hunk 3 above)
Comment (Zeko369) [reply client/components/DashboardEvent.tsx]: Take a look at [next docs](https://nextjs.org/docs/api-reference/next/link#dynamic-routes) and how the files are [named](https://nextjs.org/docs/routing/dynamic-routes) after that

Comment (timmyichen) [reply client/components/DashboardEvent.tsx]: oh, neato, TIL!

**Pull request closed without merging**